src/egit-branch.c

```diff
@@ -0,0 +1,169 @@
+#include "interface.h"
+#include "egit-repository.h"
```

Comment:
Probably you meant `egit-branch.h` here. Although it's not necessary, would also make sense to include `git2.h` (you  get it via the other includes currently.)